library/general/src/modules/Linuxrc.rb

```diff
@@ -224,6 +249,17 @@ def SaveInstallInf(root)
     publish function: :WriteYaSTInf, type: "void (map <string, string>)"
     publish function: :SaveInstallInf, type: "boolean (string)"
     publish function: :keys, type: "list <string> ()"
+    publish function: :value_for, type: "string (string)"
+
+  private
+
+    # Removes characters ignored by Linuxrc and turns all to downcase
+    #
+    # @param [String]
+    # @return [String]
+    def polish(key)
+      key.downcase.tr("-_\.", "")
```

Comment:
Backslash (`\`) is used for escaping special characters, dot (`.`) is not a special character thus the backslash is ignored:

```
irb(main):001:0> "-_\."
=> "-_."
```

Either escape the backslash (`"-_\\."`) or use single quotes (`'-_\.'`).